modules/virt-virtctl-commands.adoc

```diff
@@ -6,16 +6,33 @@
 = Virtctl client commands
 
 The `virtctl` client is a command-line utility for managing {VirtProductName}
-resources. The following table contains the `virtctl` commands used throughout
-the {VirtProductName} documentation.
+resources.
 
 To view a list of options that you can use with a command, run it with the `-h` or `--help` flag. For example:
```

Comment:
This line probably didn't get deleted after it was copied below.

Reply:
Edited the line. Please review.